package.json

```diff
@@ -42,7 +42,8 @@
     "prettier": "prettier --write \"**/*.{js,ts,tsx,md}\"",
     "prettier:check": "prettier --check \"**/*.{js,ts,tsx,md}\"",
     "test": "jest --env jsdom",
-    "types": "tsc --project tsconfig.json",
+    "emit-types": "lerna run emit-types",
+    "types": "tsc --project tsconfig.json --noEmit",
```

Comment:
How about we update these two to `types:check` and `types:emit`. Or `types` can become `ts`. Up to you.

Reply:
Did `ts:check` and `ts:emit`.